tests/selection.py

```diff
@@ -1,10 +1,18 @@
 import unittest
-import skcosmo
 from sklearn.datasets import load_boston
+import numpy as np
+
+from skcosmo.selection import FeatureCUR, FeatureFPS, SampleCUR, SampleFPS
+
+zero = 1e-8
```

Comment:
Could you use `SCREAMING_SNAKE_CASE` for global/constant values? Also, this is usually called `EPSILON` from the C `FLT_EPSILON`/`DBL_EPSILON` macros.

Reply:
Done! Snakes are screaming.